package.json

```diff
@@ -18,6 +31,7 @@
   "files": [
     "/out",
     "./l10n",
+    "./images",
```

Comment:
I agree. I can move the html. and .js files to something like `/static`? as part of this PR, or can do it as a follow up if it might step on things others are already doing?

Reply:
I'm okay either way. I think it's not fluctuating at this point, so you're welcome to do it here if you want. If it would be better focused as its own thing, that's okay too.

Reply:
I'd rather do in another PR. Having a lot of questions/issues trying to get esbuild bundles to work, so will focus on that first.

Reply:
Makes sense to me! LMK if I can help with esbuild issues. I don't know anything, but happy to be a set of 👀 if it's helpful.